plugins/ssl/multi_ssl

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+multi_ssl - Plugin to monitor CERTificate expiration on multiple services and ports
+
+=head1 CONFIGURATION
+
+  [multi_ssl_*]
+    env.services www.service.tld blah.example.net_PORT
```

Comment:
I suppose he didn't know he could use a colon.

(also, i edited your comment as i think you made the typo : color/colon)